docs/DeveloperGuide.md

```diff
@@ -371,49 +366,135 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 2a. The given index is invalid.
     * 2a1. Rapportbook shows an error message.
       Use case resumes at step 2.
-
 * 2b. No fields were specified
     * 2b1. Rapportbook shows an error message.
       Use case resumes at step 2.
-
 * 2c. Updated values is not in the right format.
     * 2c1. Rapportbook shows an error message.
       Use case resumes at step 2.
+* 2d. No index was specified.
+    * 2d-a. Rapportbook has a [contact shown](#use-case-show-contact ).
+      * Use case resumes at step 3.
+    * 2d-b. Rapportbook does not have a [contact shown](#use-case-show-contact ).
+      * 2d-b1. Rapportbook shows an error message.
```

Comment:
is this preferred over the following?
```md
2d1. 
2d2. 
    2d2a
```

Reply:
From my understanding, numbers are used to denote a sequence of steps, rather than a different section in this case here. I am not sure how to improve upon this, there must be a way.

Reply:
Thanks for the suggestion, I will update the doc to follow your suggestion.